doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -0,0 +1,189 @@
+| Field name            | Type   | Characteristics                               | Description                                                                                                                                          |
++=======================+========+===============================================+======================================================================================================================================================+
+| ``FileIdentifier``    | uint32 | Fixed-width, little-endian-encoded, unsigned. | Identifies the OTA image file at its beginning.                                                                                                      |
++-----------------------+--------+-----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``TotalSize``         | uint64 |                                               | Indicates the total size in bytes of the _entire_ file.                                                                                              |
++-----------------------+--------+-----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``HeaderSize``        | uint32 |                                               | Indicates the total size of the TLV-encoded header field.                                                                                            |
++-----------------------+--------+-----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``Header``            | TLV    | TLV encoding type and value.                  | Includes the `ota-image-header-struct` with a predefined order. See section 11.20.2.4 of the Matter specification for more information.              |
++-----------------------+--------+-----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``Payload``           | n/a    | Software image contents.                      | Includes the new software to be installed on the OTA Requestor. When using the |NCS|, this field also includes `Nordic Matter platform additions`_.  |
++-----------------------+--------+-----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. _ug_matter_overview_dfu_image_nordic:
+
+Nordic Matter platform additions
```

Comment:
I am missing information about our nRF Connect SDK solution.. which creates `matter.ota` file during the building with proper header. @Damian-Nordic what do you think, could we explain it?

Reply:
Tried to address it.

Reply:
How about the following description:
```
When building a Matter application for :doc:`nRF Connect platform <matter:nrfconnect_platform_overview>`, the Matter OTA image is one of the build :ref:`output files <app_build_output_files>`.
The default location of the software image is :file:`zephyr/matter.ota` in the build directory.
The `Payload` field of the software image includes :file:`dfu_multi_image.bin` file, which is another :ref:`output file <app_build_output_files_other>` of the build process.

The :file:`dfu_multi_image.bin` file is an archive file that includes a CBOR manifest and a collection of user-selected update components.
The CBOR manifest contains identifiers and sizes of the included update components to allow a user to successfully unpack the archive.
The default update components are firmware images for all MCU cores.

During the Matter OTA firmware updade process, the downloaded Matter OTA image is saved to external flash using the :ref:`lib_dfu_multi_image` and :ref:`lib_dfu_target` libraries.
The :ref:`lib_dfu_multi_image` library uses `zcbor`_ for parsing the CBOR header.
Applying the software image requires rebooting to :ref:`MCUBoot <mcuboot:mcuboot_ncs>` bootloader which installs the new firmware.
```